mmv1/api/resource/examples.go

```diff
@@ -176,6 +178,32 @@ func (e *Examples) UnmarshalYAML(n *yaml.Node) error {
 }
 
 func (e *Examples) SetHCLText() {
+	e.DocumentationHCLText = ExecuteHCL(e)
+
+	copy := e
```

Comment:
It is better to keep the comments in https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/provider/terraform/examples.rb#L204-L220?

Reply:
yes, added them in